src/main/java/org/openrewrite/maven/MavenMojoProjectParser.java

```diff
@@ -800,4 +804,22 @@ private void logDebug(MavenProject mavenProject, String message) {
     private static <E extends Throwable> E sneakyThrow(Throwable e) throws E {
         return (E) e;
     }
+
+    private static ResolvedGroupArtifactVersion createResolvedGAV(MavenProject project, MavenExecutionContextView ctx) {
+        return new ResolvedGroupArtifactVersion(
+                ctx.getLocalRepository().getUri(),
+                project.getGroupId(),
+                project.getArtifactId(),
+                project.getVersion(),
+                project.getVersion().endsWith("-SNAPSHOT") ? null : project.getVersion()
+        );
+    }
+
+    private static Pom createPom(ResolvedGroupArtifactVersion gav, MavenProject project) {
+        Pom.PomBuilder builder = Pom.builder()
+                .gav(gav)
+                ;
+        // TODO: complete mapping
+        return builder.build();
```

Comment:
I guess this is why it's still a WIP; let me know your plans and whether you'd need any help.